bin/configure-monitoring-secret.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+# This script creates SSM parameter for storing integration URL for incident management
+# services. Script creates new SSM attribute or updates existing. 
+#
+# Positional parameters:
+#   APP_NAME (required) – the name of subdirectory of /infra that holds the
+#     application's infrastructure code.
+#   ENVIRONMENT is the name of the application environment (e.g. dev, staging, prod)
+#   URL is the url for te integration endpoint for external incident management
```

Comment:
nit: updating the comment to reflect the name of the variable below

Reply:
done and updated a description a bit